### ISSUE TITLE
V8 - Grid - allow editing image alt text

### ISSUE THREAD

**Issue by nathanwoulfe** [umbraco/Umbraco-CMS#5596]
Quick win - when adding an image control on the grid, editors can optionally provide alt text when selecting their image. What they can't do is edit that alt text without removing the image, reselecting, and providing new text.

Can be easily remedied by adding an additional input to the media grid view, below/above the existing one for the image caption.

Will have pull request shortly.

**Comment by nathanwoulfe** [comment]
Actually, not as quick a win on 8 as 7 - v8 stores the alt text value as the image caption, which doesn't make sense as they maybe two different values. Better approach would be to allow editing these separately, but render the caption as the alt text if no value is given for the latter.

Shouldn't assume that the alt text is appropriate for use as a caption, but should be ok to do the opposite.

@nielslyngsoe appreciate your thoughts before I touch anything...

**Comment by nielslyngsoe** [comment]
@nathanwoulfe I completely agree with you. Its two different things so let's embrace that.

**Comment by osigas** [comment]
Think this is a separate issue, but should the focal point also be treated similarly? That doesn't seem to be being pulled through on to the Media value object as far as I can see.

**Comment by arknu** [comment]
@craigs100 I would assume so. I'm facing the same issue. Whoever thought that having alt text pop up in the caption field was a good idea?

**Comment by arknu** [comment]
@nul800sebastiaan Why is this issue still open a year after a PR was submitted? Just ran into this issue on a client site. 
I'd work on fixing this, but since there already is a PR, I don't see what I can do to get this fixed...

**Comment by TimWensink** [comment]
I would like to put this issue back on the radar.

The image caption should be a seperate field instead of it being the alt-tag. Content editors should be able to add different values here. The alt-tag is not always what you need/want to show as a caption  and vice-versa, and with accessibillity and SEO in mind, I think this needs to be changed.

I have run into this issue on multiple projects and one of the clients has reached out to me with questions about it.
Can you give us a (rough) estimate on when this will be addressed and fixed?

**Comment by nathanwoulfe** [comment]
@TimWensink I agree (and I agreed when I got the notification, before I realised this was my issue + pr)

It's super stale give it's 2 years old, but I'll try to find a few minutes to run the pr against the latest v8/contrib and see what happens...

Would be good to get it merged, seems like a reasonable change to me (obviously, I made the initial change 😁)

**Comment by TimWensink** [comment]
@nathanwoulfe Glad to see your (quick) reply! Thank's for looking into this. Hopefully it will be an easy merge. 😁

**Comment by nathanwoulfe** [comment]
I've redone the initial PR - old one was super stale, was easier to toss it out and redo. Changes are in #9681.

ping @TimWensink @craigs100 @arknu @nielslyngsoe

**Comment by TimWensink** [comment]
@nathanwoulfe I see some conversation and some additional changes made in the pull-request, but now it's been quiet for 8 days.
Do you have an idea when this will be merged into a hotfix, and when we can expect this to roll out?

Want to make sure this doesn't get lost/forgotten like it has with your initial work in 2019. That would be ashame.
Some clients and project managers are eager to use this and would like to know when we are able to roll this out. 😁

**Comment by nathanwoulfe** [comment]
@TimWensink the new PR will need to be reviewed before merging - if I ping @nul800sebastiaan @mikecp @nielslyngsoe @abjerner @OwainWilliams, one of these fine characters might have the time to review and merge...

**Comment by nul800sebastiaan** [comment]
Fixed for 8.12 in https://github.com/umbraco/Umbraco-CMS/pull/9681
